models/Asset/Dao.php

```diff
@@ -158,7 +159,7 @@ public function updateWorkspaces()
     public function updateChildPaths($oldPath)
     {
         //get assets to empty their cache
-        $assets = $this->db->fetchCol('SELECT id FROM assets WHERE path LIKE ' . $this->db->quote($oldPath . '%'));
+        $assets = $this->db->fetchCol('SELECT id FROM assets WHERE path LIKE ' . $this->db->quote(Db::get()->escapeLike($oldPath . '%')));
```

Comment:
Whenever possible we shouldn't use the static getter, better use `$this->db->escapeLike()` instead if possible.